invenio_rdm_records/services/tasks.py

```diff
@@ -41,15 +41,20 @@
 @shared_task(ignore_result=True)
 def update_expired_embargos():
     """Lift expired embargos."""
+    current_app.logger.debug("Updating expired embargoes")
```

Comment:
how does the `current_app.logger` knows to which job the logging information has to be added?

Reply:
that would mean if i create a Job to wrap a celery task, i don't have to do anything and it should work?

Reply:
ok, then i am doing something wrong, because it doesn't show up on `administrations/runs/RUN_ID`.

Reply:
i have a problem in replicating the behavior, i wrote you more on discord